release-resources/release-strategy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: e2e-fbc-strategy
+  namespace: 
+spec:
+  bundle: 'quay.io/hacbs-release/pipeline-fbc-release:main'
+  params:
+    - name: fromIndex
+      value: 'quay.io/scoheb/fbc-index-testing:latest'
+    - name: targetIndex
+      value: 'quay.io/scoheb/fbc-target-index-testing:latest'
+    - name: binaryImage
+      value: 'registry.redhat.io/openshift4/ose-operator-registry:v4.12'
+    - name: requestUpdateTimeout
+      value: '420'
+    - name: buildTimeoutSeconds
+      value: '480'
+  pipeline: fbc-release
+  policy: fbc-policy
+  serviceAccount: release-service-account
```

Comment:
when is the SA created? Are we saying this and the policy already exist on stage and if not we just fail?

Reply:
as part of setting up the environment before the test  as agreed with @scoheb